### PR TITLE
Fix/find

### PR DESCRIPTION
**Pull request by jinhanjinhan**
**Summary of changes:**
1. Update display messages for clarity
2. Add logging to existing code
3. Remove NameContainsKeywordsPredicate class (as it is replaced by FieldContainsKeywordsPredicate class)
4. Add field identifier to FieldContainsKeywordsPredicate class, which is used for checking in the equals() method
5. Add equals() and toString() method to CreateTutorialCommand and CloseTutorialCommand
6. Add test cases

### CODE REVIEW

File: src/test/java/seedu/address/logic/parser/CloseTutorialCommandParserTest.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        parser = new CloseTutorialCommandParser();
+    }
+
+    @Test
+    public void parse_validInput_returnsCloseTutorialCommand() throws Exception {
+        String args = " tut/Math";
+        CloseTutorialCommand command = parser.parse(args);
+
+        // Check that the parser returns a CloseTutorialCommand with the correct parameters
+        assertEquals(new CloseTutorialCommand(new Tutorial("Math")), command);
```
Comment (gladwintan) [inline src/test/java/seedu/address/logic/parser/CloseTutorialCommandParserTest.java:31]: Could we use assertParseSuccess() method that is provided in CommandParserTestUtil instead?

File: src/test/java/seedu/address/logic/parser/CloseTutorialCommandParserTest.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    }
+
+    @Test
+    public void parse_missingTutorialPrefix_throwsParseException() {
+        String args = "Math";
+        ParseException exception = assertThrows(ParseException.class, () -> parser.parse(args));
+
+        // Verify that the exception message contains the expected usage message
+        assertTrue(exception.getMessage().contains(CloseTutorialCommand.MESSAGE_USAGE));
```
Comment (gladwintan) [inline src/test/java/seedu/address/logic/parser/CloseTutorialCommandParserTest.java:40]: Could we use assertParseFailure() here?

**Review by gladwintan — COMMENTED** [review]
Looks good

**Review by Slipperybara — APPROVED** [review]
LGTM, should we all standardise our logging for start of commands and end of commands?

File: src/test/java/seedu/address/logic/parser/CloseTutorialCommandParserTest.java (hunk 1 above)
Comment (jinhanjinhan) [reply src/test/java/seedu/address/logic/parser/CloseTutorialCommandParserTest.java:31]: Ahhh right, thanks for the good spot

File: src/test/java/seedu/address/logic/parser/CloseTutorialCommandParserTest.java (hunk 2 above)
Comment (jinhanjinhan) [reply src/test/java/seedu/address/logic/parser/CloseTutorialCommandParserTest.java:40]: Yep we can, I will change it in my code and push it again, thanks for the spot!

**Review by gladwintan — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/FindCommand.java [hunk 3]
```diff
@@ -66,6 +84,7 @@ public FindCommand(List<Predicate<Person>> personPredicates,
 
     @Override
     public CommandResult execute(Model model) {
+        logger.info(" - Running execute(Model model) in " + FindCommand.class);
```
Comment (amitha2210) [inline src/main/java/seedu/address/logic/commands/FindCommand.java:87]: Good use of logging throughout!

**Review by amitha2210 — APPROVED** [review]
LGTM

**Pull request merged**